src/Adapter/Cart/CartPresenter.php

```diff
@@ -198,8 +201,12 @@ public function addCustomizedData(array $products, Cart $cart)
         }, $products);
     }
 
-    public function present(Cart $cart)
+    public function present(ObjectModel $cart)
     {
+        if (!is_a($cart, 'Cart')) {
```

Comment:
and probably this is not needed because PHP will throw an error if `$cart` is not instance of `Cart`?